debugger/src/tpage.c

```diff
@@ -142,19 +142,18 @@ void tpage_pack_widgets(gboolean tabbed)
 	GList *children = gtk_container_get_children(GTK_CONTAINER(tab_target));
 	if (children)
 	{
-		int i;
-
-		oldroot = (GtkWidget*)children->data;
+		GtkWidget **widget_ref = NULL;
```

Comment:
I'd rather have this declared in the loop

Reply:
Done in 516277c184f2019e2e504f1d598b008cb6695358.